README.md

```diff
@@ -13,97 +13,129 @@ urlFragment: functions-quickstart-python-azd
 
 # Azure Functions Python HTTP Trigger using AZD
 
-This repository contains a Azure Functions HTTP trigger quickstart written in Python and deployed to Azure using Azure Developer CLI (AZD). This sample uses managed identity and a virtual network to insure it is secure by default. 
+This repository contains an Azure Functions HTTP trigger quickstart written in Python and deployed to Azure using Azure Developer CLI (AZD). This sample uses managed identity and a virtual network to insure it's secure by default. 
 
-## Getting Started
+## Prerequisites
 
-### Prerequisites
-
-1) [Python 3.11](https://www.python.org/) 
-2) [Azure Functions Core Tools](https://learn.microsoft.com/azure/azure-functions/functions-run-local?tabs=v4%2Cmacos%2Ccsharp%2Cportal%2Cbash#install-the-azure-functions-core-tools)
-3) [Azure Developer CLI (AZD)](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd)
-4) [Visual Studio Code](https://code.visualstudio.com/) - Only required if using VS Code to run locally
++ [Python 3.11](https://www.python.org/)
++ [Azure Functions Core Tools](https://learn.microsoft.com/azure/azure-functions/functions-run-local#install-the-azure-functions-core-tools)
++ [Azure Developer CLI (AZD)](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd)
++ To use Visual Studio Code to run and debug locally:
+  + [Visual Studio Code](https://code.visualstudio.com/) 
+  + [Azure Functions extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions)
 
 ### Get repo on your local machine
-Run the following GIT command to clone this repository to your local machine.
-```bash
-git clone https://github.com/Azure-Samples/functions-quickstart-python-azd.git
-```
 
-## Run on your local environment
+You can download this project template in one of these ways:
```

Comment:
I like this but I think it's making it a little too long.  I would show only 1 or possibly 2 methods.  To me either azd init (no git) or fork and clone.